### PR TITLE
method to add users in bulk added

### PR DESCRIPTION
**Pull request by MSlootwegVIKTOR**

### CODE REVIEW

File: viktor_dev_tools/tools/subdomain.py [hunk 1]
```diff
@@ -582,12 +602,8 @@ def upload_database_from_local_folder(self, source_folder: str, filename: str):
         validate_root_entities_compatibility(database_dict["entities"], destination_root_entities)
 
         print("Successfully validated database compatibility. Removing children...")
-        click.confirm(
```
Comment (jjvandenberg1) [inline viktor_dev_tools/tools/subdomain.py]: why remove this?

File: viktor_dev_tools/tools/subdomain.py [hunk 2]
```diff
@@ -35,6 +35,22 @@ class EntityDict(TypedDict, total=False):
     parent_entity_type: int
 
 
+class UserDict(TypedDict, total=False):
```
Comment (jjvandenberg1) [inline viktor_dev_tools/tools/subdomain.py]: please rename this class, UserDict is already in use

Comment (jjvandenberg1) [reply viktor_dev_tools/tools/subdomain.py]: please make sure that running dev-tools locally from cli.py still works, it's throwing the following error:
ImportError: cannot import name 'UserDict' from 'viktor_dev_tools.tools.subdomain' (C:\WORK\PYTHON\viktor-apps\development-tools\venv\Lib\site-packages\viktor_dev_tools\tools\subdomain.py)

File: viktor_dev_tools/cli.py [hunk 3]
```diff
@@ -5,12 +5,14 @@
 """
 import subprocess
```
Comment (jjvandenberg1) [inline viktor_dev_tools/cli.py:6]: please make sure to add any new functions to click, such that the users is prompted all options

**Review by jjvandenberg1 — COMMENTED** [review]
could not test fully since it is throwing some critical errors

File: viktor_dev_tools/tools/subdomain.py (hunk 1 above)
Comment (MSlootwegVIKTOR) [reply viktor_dev_tools/tools/subdomain.py]: Should not have happened. Added it again.

File: viktor_dev_tools/tools/subdomain.py (hunk 2 above)
Comment (MSlootwegVIKTOR) [reply viktor_dev_tools/tools/subdomain.py]: `UserDict` refactored to `ViktorUserDict`

Strange, it is working on my PC. Could you check again?

File: viktor_dev_tools/cli.py (hunk 3 above)
Comment (MSlootwegVIKTOR) [reply viktor_dev_tools/cli.py:6]: Added

**Pull request merged**